Mergin/validation.py

```diff
@@ -485,3 +493,5 @@ def warning_display_string(warning_id, url=None):
         return f"You don't have permission to change the configuration of this project. <a href='{url}'>Reset the configuration</a> to be able to sync data changes."
     elif warning_id == Warning.EDITOR_DIFFBASED_FILE_REMOVED:
         return f"You don't have permission to remove this layer. <a href='{url}'>Reset the layer</a> to be able to sync changes."
+    elif warning_id == Warning.PROJECT_HOME_PATH:
+        return "QGIS Project Home Path is specified. <a href='fix_project_home_path'>Fix the issue.</a>"
```

Comment:
Silently reset the project home, we could warm the user just in case

```suggestion
        return "QGIS Project Home Path is specified. <a href='fix_project_home_path'>Quick fix the issue. (This will unset project home) </a>"
```